tests/testthat/test-vec_quantiles.R

```diff
@@ -0,0 +1,25 @@
+test_that("vec_quantiles error types", {
+  expect_error(vec_quantiles(1:10, 1:4 / 5), "matrix")
+  expect_error(
+    vec_quantiles(matrix(1:20, 5), -1:4 / 5),
+    "`quantile_levels` must be a number between 0 and 1"
+  )
```

Comment:
```suggestion
  expect_snapshot(
    error = TRUE,
    vec_quantiles(matrix(1:20, 5), -1:4 / 5)
  )
```

Will just do one of these, but we now prefer snapshot tests for errors over `expect_error()`!